src/dealDashboard/dealMenubar/dealMenubar.scss

```diff
@@ -1,36 +1,48 @@
 @import "src/resources/elements/primeDesignSystem/variables";
 
 deal-menubar {
-  display: grid;
-  grid-template-columns: auto auto auto;
-  align-items: center;
+  display: flex;
   justify-content: end;
-  grid-column-gap: 10px;
-  grid-row-gap: 20px;
 
-  i.fas {
-    font-size: 20px;
+  .dealActionButtons {
+    display: grid;
+    grid-template-columns: auto auto;
+    align-items: center;
+    justify-content: end;
+    grid-column-gap: 10px;
+    grid-row-gap: 20px;
+
+    margin-left: 30px;
```

Comment:
before: margin-right from the Share button.
after: margin-left from [Edit Cancel]